static/js/contribute.js

```diff
@@ -0,0 +1,42 @@
+//Make the DIV element draggagle:
+dragElement(document.getElementById(("contribute-box")));
+
+function dragElement(elmnt) {
+  var pos1 = 0, pos2 = 0, pos3 = 0, pos4 = 0;
+  if (document.getElementById(elmnt.id + "-tabs")) {
```

Comment:
Since this element is being queried repeatedly, it should be looked up once and stored so we don't need to duplicate the element ID construction string on two subsequent lines:

```
var tabElement = document.getElementById(elmnt.id + "-tabs");
if (tabElement) {
    tabElement.onmousedown …
}
```